docs/markdown/Contributing.md

```diff
@@ -18,6 +18,39 @@ Github](https://github.com/mesonbuild/meson/pulls). This causes them
 to be run through the CI system. All submissions must pass a full CI
 test run before they are even considered for submission.
 
+## Acceptance and merging
+
+The kind of review and acceptance any merge proposal gets depends on
+the changes it contains. All pull requests must be reviewed and
+accepted by someone with commit rights who is not the original
+submitter. Merge requests can be roughly split into two different
+categories.
+
+The first consists of merges that don't change any functionality,
+fixes to the CI system and bug fixes that have added regression tests
```

Comment:
Does #3195 fall into "bug fixes that have added regression tests"?

Reply:
I would say so, yes.